### DISCUSSION TITLE
Arrow keys by default should ALWAYS increment by the smallest amount possible on value sliders.

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#28672] [Ideas]
Currently, there are a few sliders (background dim, local beatmap offset, etc.) that increment by 5s (and tablet area 12x7 for some reason.) In all these cases, the arrow keys should just increment by 1, thats the main reason someone would even use the arrow keys. If the user wants to move the value by more than 1 they can use either their mouse, or maybe using the vertical arrows would increase the increment by 10x. 
A lot of sliders already follow this smallest increment rule, so it doesn't make much sense to have arbitrary increments on random sliders.

**Comment by M-ael** [comment]
I agree with this suggestion. Make all slider increments consistent. 
But - just like with any change - please make it customizeable. Either with an additional key (ex: _Ctrl_ + Alt + Up/Down) or a setting to change the size of the increment. I myself like to use an increment of 5 and don't like to have to click multiple times when I can do it in 1 click in osu!stable.
Just make it customizeable, that's best - like with any change.
